### DISCUSSION TITLE
Column 관련 질문이 있습니다

### DISCUSSION THREAD

**Discussion by fenoteryn** [bosskmk/pluto_grid#205] [Q&A]
removeRow나 appendRow와 같은 함수로 Row값을 수정할 수 있는데 Column도 그런 기능이 있나요? 버튼을 누르면 Column값이 바뀌거나 그리드를 새로 그려서 Column을 업데이트하고 싶습니다. 방법 알 수 있을까요?

**Comment by bosskmk** [comment]
컬럼은 변경하는 기능이 없습니다. column과 row 의 값을 직접 변경하고 `stateManager.addListeners()` 를 호출하여 그리드의 상태를 업데이트 할 수 있습니다.

**Reply by fenoteryn** [reply]
stateManager.addListners()를 어떤 상황에서 호출해야 하는지 알려 주실 수 있나요? column이 유동적인 그리드를 작성하고 싶습니다..

**Reply by fenoteryn** [reply]
아니면 그리드를 삭제하고 재작성해서 column이 바뀌는 방법이 가능할까요?
